docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,341 @@
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Trial license
+|https://trialrequest.hazelcast.com/[Hazelcast website]
+
+|Docker image for the full Hazelcast distribution
+|xref:get-started-enterprise.adoc[Install Hazelcast Enterprise]
+
+|===
+
+== Step 1. Configure the Enterprise Member
+
+Before a member can authenticate a client, the member must be configured with security enabled and some credentials to ask the client for.
+
+Create a `hazelcast.yaml` (or `xml`) configuration file with the following content and place it in the `~/config` directory on your local machine.
+
+[tabs]
+====
+YAML::
++
+--
+
+[source,yaml]
+----
+hazelcast:
+  cluster-name: hello-world
+  license-key: <your license key> <1>
+  security:
+    enabled: true <2>
+    realms:
+      - name: passwordRealm <3>
+        identity:
+          username-password: <4>
+            username: member1
+            password: s3crEt
+  member-authentication:
+    realm: passwordRealm <5>
+----
+<1> Replace the `<your license key>` placeholder with your Hazelcast Enterprise license key.
+<2> Set to `true` to enable security so that the clients can be authenticated by the member.
+<3> Name of the security realm to be passed to the authentication configuration; it is set to `passwordRealm` in this example for convenience.
+<4> Identity configuration element to set the credentials (username and password) to be used by the member for client authentication.
+<5> Name of the security realm to be used by the member for client authentication.
+--
+
+XML::
++
+[source,xml]
+----
+<hazelcast>
+    <cluster-name>hello-world</cluster-name>
+    <license-key>your license key</license-key> <1>
+    <security enabled="true"> <2>
+        <realms>
+            <realm name="passwordRealm"> <3>
+                <identity>
+                    <username-password username="member1" password="s3crEt" /> <4>
+                </identity>
+            </realm>
+        </realms>
+        <member-authentication realm="passwordRealm" /> <5>
+    </security>
+</hazelcast>
+----
+<1> Replace the `<your license key>` placeholder with your Hazelcast Enterprise license key.
+<2> Set to `true` to enable security so that the clients can be authenticated by the member.
+<3> Name of the security realm to be passed to the authentication configuration; it is set to `passwordRealm` in this example for convenience.
+<4> Identity configuration element to set the credentials (username and password) to be used by the member for client authentication.
+<5> Name of the security realm to be used by the member for client authentication.
+====
+
+With this configuration, you tell the member to accept connections only from the clients
+with a username of `member1` and a password of `s3crEt`.
+
+== Step 2. Start the Member
+
+Run the following command.
+
+[source,shell,subs="attributes+"]
+----
+docker run \
+    --net=host \
```

Comment:
```suggestion
    -p=5701:5701 \
```

The host network is supported only in Linux: https://docs.docker.com/network/host/

We usually just expose port 5701 in other tutorials, so I propose to do the same here instead.